GetJobber/atlantis
# fix(hooks): Update package file [JOB-32207]

## Motivations
Removes the need to specify the `types` root by ensuring that `hooks` uses the same `@types/react` version as the root project

## Testing

- Run `npm install` and make sure the build works.
- Run specs

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
